app/Http/Controllers/V2/Auditable/UpdateAuditableStatusController.php

```diff
@@ -78,12 +78,15 @@ private function canChangeSitePolygonStatusTo($sitePolygon, $status)
                 return false;
             }
 
-            $criteriaList = array_filter($criteriaList, function ($criteria) {
-                return $criteria['criteria_id'] !== PolygonService::ESTIMATED_AREA_CRITERIA_ID;
+            $criteriaArray = $criteriaList->toArray();
+
+            $criteriaArray = array_filter($criteriaArray, function ($criteria) {
+                return $criteria['criteria_id'] !== PolygonService::ESTIMATED_AREA_CRITERIA_ID &&
+                    $criteria['criteria_id'] !== PolygonService::DATA_CRITERIA_ID;
```

Comment:
I'll approve because this is just a 🎨 comment, but I'd find it slightly more readable (not sure about performance implications; there probably aren't any, or they're minor if there are) to filter and then fetch the array (assuming $criteriaList is a laravel collection). And either way, an arrow function would be appropriate here: 
```
$criteriaArray = $criteriaList
    ->filter(fn ($criteria) => 
        $criteria['criteria_id'] !== PolygonService::ESTIMATED_AREA_CRITERIA_ID &&
        $criteria['criteria_id'] !== PolygonService::DATA_CRITERIA_ID
    )->toArray()
````